setup.py

```diff
@@ -1,14 +1,29 @@
+import os.path
+
+import codecs
 from setuptools import find_packages, setup
 
-with open("README.md", "r") as file:
-    long_description = file.read()
+
+def read(rel_path):
+    here = os.path.abspath(os.path.dirname(__file__))
+    with codecs.open(os.path.join(here, rel_path), "r") as fp:
+        return fp.read()
+
+
+def get_version(rel_path):
+    for line in read(rel_path).splitlines():
+        if line.startswith("__version__"):
+            delimiter = '"' if '"' in line else "'"
+            return line.split(delimiter)[1]
```

Comment:
Another approach you could use (I don't know if it's better or worst, just different, I think) is to write something like:
```return line.replace('"', "").replace("'", "")```
or maybe
```
import re

...

return re.sub(r"[\"\']", "", line)
```